src/main/java/address/controller/StatusBarFooterController.java

```diff
@@ -57,27 +57,28 @@ private void syncSchedulerOnTimeOut() {
 
     /**
      * Initializes the status bar
-     *
-     * The given config object should have set updatedInterval
-     *
-     * @param config
+     * @param updateInterval The sync period
+     * @param saveLocation The location to save the file for storing Addressbook contacts.
      */
-    public void initStatusBar(Config config) {
-        updateWithConfigValues(config);
+    public void init(long updateInterval, String saveLocation) {
+        initSyncTimer(updateInterval);
+        initStatusBar();
+        initSaveLocationLabel(saveLocation);
+    }
+
+    private void initSaveLocationLabel(String saveLocation) {
+        secondaryStatusBarLabel.setText(SAVE_LOC_TEXT_PREFIX + saveLocation);
```

Comment:
can call `updateSaveLocationDisplayHere`? So the format will always be the same.